### PR TITLE
Update working-with-the-nuget-registry.md

### PR DESCRIPTION
**Pull request by zammitjohn**
RepositoryURL is a project file config

### What's being changed (if available, include any code snippets, screenshots, or gifs):
I'd like to point out a small discrepancy in the documentation regarding the configuration of the `RepositoryURL`. It is currently stated that it should be set in the `nuget.config` file; however, this is not accurate. The correct place to specify the `RepositoryURL` is within the `.csproj` file of the project.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

File: content/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ The NuGet registry stores packages within your organization or personal account,
...
 
-If you specify a `RepositoryURL` in your `nuget.config` file, the published package will automatically be connected to the specified repository. For more information, see "[AUTOTITLE](/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#publishing-a-package-using-a-nugetconfig-file)." For information on linking an already-published package to a repository, see "[AUTOTITLE](/packages/learn-github-packages/connecting-a-repository-to-a-package)."
+If you specify a `RepositoryURL` in your project's file, which ends in _.csproj_, the published package will automatically be connected to the specified repository. For more information, see "[AUTOTITLE](/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#publishing-a-package-using-a-nugetconfig-file)." For information on linking an already-published package to a repository, see "[AUTOTITLE](/packages/learn-github-packages/connecting-a-repository-to-a-package)."
```
Comment (thyeggman) [inline content/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry.md:112]:
```suggestion
If you specify a `RepositoryURL` in your project's _.csproj_ file, the published package will automatically be connected to the specified repository. For more information, see "[AUTOTITLE](/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#publishing-a-package-using-a-nugetconfig-file)." For information on linking an already-published package to a repository, see "[AUTOTITLE](/packages/learn-github-packages/connecting-a-repository-to-a-package)."
```

**Review by thyeggman — COMMENTED** [review]
Nice catch! I appreciate trying to match the other references to the `.csproj` in this file, but I think we can just use the simpler wording in this case 😄

**Review by thyeggman — APPROVED** [review]

**Review by nguyenalex836 — APPROVED** [review]

**Comment by Githubant123** [comment]
Update it

**Comment by Githubant123** [comment]
Update it

**Comment by Githubant123** [comment]
Update it

**Pull request merged**